.github/workflows/create_release.yml

```diff
@@ -0,0 +1,73 @@
+name: Create release with _freeze
+
+on:
+  workflow_dispatch:
+  # Comment out before merging!!
+  push:
+    branches:
+      - py/release-workflow
```

Comment:
This PR is marked as draft because these lines should be _removed_ (not commented, lol) before merging. I added them just so that we can see if the workflow runs correctly before merging.

Reply:
Keep an eye on the workflow run here: https://github.com/TuringLang/docs/actions/runs/11644683602/job/32426851893?pr=544